spec/integration/studio/v1/flow/execution_spec.rb

```diff
@@ -104,6 +106,7 @@
     }.to raise_exception(Twilio::REST::TwilioError)
 
     values = {'To' => '+15558675310', 'From' => '+15017122661', }
+    headers = {}
```

Comment:
Seeing a lot of these that are unused.

Reply:
Pushing update to add an additional check when rendering these